packages/react-redux-subspace/src/components/SubspaceProvider.js

```diff
@@ -25,22 +58,40 @@ class SubspaceProvider extends React.PureComponent {
     }
 }
 
+SubspaceLegacyContextOutput.childContextTypes = {
+    store: PropTypes.object
+}
+
+class SubspaceProvider extends React.Component {
+    makeSubspace = memoizeOne((mapState, namespace, subspaceDecorator, store) => {
```

Comment:
We actually get a perf boost from this change as we are only going to recalculate the subspace if the params to create it change.  Previously we would recalculate it any time `getChildContext` was called (which I believe is on every render).